idseq_dag/util/command.py

```diff
@@ -127,7 +127,6 @@ def __exit__(self, *args):
             # TODO: Do we need to join the tail subproc after killing it?
             self.tail_subproc.kill()
 
-
```

Comment:
Nit: Don't remove empty line. Looks like we have convention of 2 newlines between methods.

Reply:
Done